docs-web/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -95,6 +95,10 @@ spec:
      - A list of routes.
      - `[]route <#virtualserver-route>`_
      - No
+   * - ``class``
+     - Specifies which Ingress controller must handle the VirtualServer resource. Set to `nginx` to make NGINX Ingress controller handle it.
```

Comment:
The second sentence assumes that other Ingress Controllers can handle VirtualServer resources (not true) and that our Ingress Controller has the `nginx` class (not always true). That's why I suggest removing it here and in the VSR doc:
```suggestion
     - Specifies which Ingress controller must handle the VirtualServer resource.
```